### ISSUE TITLE
LCP load time exceeds Google's acceptable range

### ISSUE THREAD

**Issue by Popolechien** [kiwix/web#125]
Apparently this penalizes users and site ranking
https://developers.google.com/speed/pagespeed/insights/?hl=en&url=https%3A%2F%2Fwww.kiwix.org%2F

The good news is that Google suggests a couple of solutions for the most problematic issues, so maybe there's hope.

**Comment by kelson42** [comment]
This is not a super critical problem, but I would recommend to consider using https://wordpress.com/ as a hoster.

**Comment by martin8032** [comment]
I don't think that our hosting provider is the main problem. And using wordpress.com instead of a self-hosted WordPress installation might cause other issues. Website performance is a complex topic – there is no simple solution for this.

**Comment by kelson42** [comment]
@martin8032 What is the root cause of the problem?

**Comment by martin8032** [comment]
I don't think that there is one single root cause but many factors which add up. Kinsta has a comprehensive guide on this: https://kinsta.com/de/lernen/wordpress-beschleunigen/ According to Google PageSpeed, render-blocking resources are currently the main issue. I have experimented with some WordPress plugins for performance optimization, but without success. This is beyond my current technical knowledge and would require a developer. Or we go back to a very basic website with no Divi page builder, no plugins, no nothing – like this demo site: https://developers.google.com/speed/pagespeed/insights/?hl=en&url=https%3A%2F%2Fdemo.metoki.ch%2F&tab=mobile

**Comment by kelson42** [comment]
@martin8032 Thank you. We will have to do the necessary somehow to identify exactly the root cause(s) of the problem. I don't see any solution path without doing that part of the work. I probably will have to do it myself.

**Comment by kelson42** [comment]
Still a problem with new Web sites.

**Comment by rgaudin** [comment]
Duplicate of #271 now
